wgs.yaml

```diff
@@ -0,0 +1,208 @@
+sigs: []
```

Comment:
Should we add all information related to wg under `/community/wgs` or we want to follow k8s structure way where we add folders to `/community`

Reply:
You don't add anything. You ***only*** make changes to the `wgs.yaml` file. PTAL at the `generator/README.md` for more information on how it works

Reply:
Yeah, I got it.
I am just thinking do we want to modify generate script to follow steps, introduced here: https://github.com/kubeflow/community/blob/master/wgs/wg-lifecycle.md.

Reply:
I've edited that part [here](https://github.com/kubeflow/community/blob/618908e73928afb827649dc55a3131b749564820/wgs/wg-lifecycle.md#github) to follow new instructions